src/preconditions/index.ts

```diff
@@ -1,13 +1,13 @@
-export { CorePrecondition as ClientPermissions } from './ClientPermissions';
-export { CorePrecondition as Cooldown } from './Cooldown';
-export { CorePrecondition as DMOnly } from './DMOnly';
-export { CorePrecondition as Enabled } from './Enabled';
-export { CorePrecondition as GuildNewsOnly } from './GuildNewsOnly';
-export { CorePrecondition as GuildNewsThreadOnly } from './GuildNewsThreadOnly';
-export { CorePrecondition as GuildOnly } from './GuildOnly';
-export { CorePrecondition as GuildPrivateThreadOnly } from './GuildPrivateThreadOnly';
-export { CorePrecondition as GuildPublicThreadOnly } from './GuildPublicThreadOnly';
-export { CorePrecondition as GuildTextOnly } from './GuildTextOnly';
-export { CorePrecondition as GuildThreadOnly } from './GuildThreadOnly';
-export { CorePrecondition as NSFW } from './NSFW';
-export { CorePrecondition as UserPermissions } from './UserPermissions';
+export { ClientPermissionsPrecondition as ClientPermissions, UserPermissionsPreconditionContext } from './ClientPermissions';
+export { CooldownPrecondition as Cooldown, CooldownContext } from './Cooldown';
+export { DMOnlyPrecondition as DMOnly } from './DMOnly';
+export { EnabledPrecondition as Enabled } from './Enabled';
+export { GuildNewsOnlyPrecondition as GuildNewsOnly } from './GuildNewsOnly';
+export { GuildNewsThreadOnlyPrecondition as GuildNewsThreadOnly } from './GuildNewsThreadOnly';
+export { GuildOnlyPrecondition as GuildOnly } from './GuildOnly';
+export { GuildPrivateThreadOnlyPrecondition as GuildPrivateThreadOnly } from './GuildPrivateThreadOnly';
+export { GuildPublicThreadOnlyPrecondition as GuildPublicThreadOnly } from './GuildPublicThreadOnly';
+export { GuildTextOnlyPrecondition as GuildTextOnly } from './GuildTextOnly';
+export { GuildThreadOnlyPrecondition as GuildThreadOnly } from './GuildThreadOnly';
+export { NSFWPrecondition as NSFW } from './NSFW';
+export { UserPermissionsPrecondition as UserPermissions } from './UserPermissions';
```

Comment:
Why were they renamed?

Reply:
Because CorePreconditions.CooldownPrecondition is redundant

Reply:
No, I'm talking about them being renamed from `CorePrecondition`.

Reply:
Oh that...I actually don't remember